proxy-node-translator/src/main/java/uk/gov/ida/notification/translator/resources/HubResponseTranslatorResource.java

```diff
@@ -50,14 +57,40 @@ public Response hubResponse(HubResponseTranslatorRequest hubResponseTranslatorRe
         final X509Certificate encryptionCertificate = X_509_CERTIFICATE_FACTORY.createCertificate(hubResponseTranslatorRequest.getConnectorEncryptionCertificate());
 
         final org.opensaml.saml.saml2.core.Response eidasResponse = eidasResponseGenerator.generate(hubResponseContainer, encryptionCertificate);
+
+        String hashedEidasDetails = hashResponseDetails(
+                translatedHubResponse.getPid(),
+                translatedHubResponse.getAttributes().getFirstName().getValue(),
+                combineAttributeValues(translatedHubResponse.getAttributes().getMiddleNames()),
+                combineAttributeValues(translatedHubResponse.getAttributes().getSurnames()),
+                translatedHubResponse.getAttributes().getDateOfBirth().getValue().format(DateTimeFormatter.ofPattern("YYYY-MM-dd"))
```

Comment:
This could be brittle

Reply:
What would improve it?

Reply:
I've tidied this up and moved onto multiple lines. The DateOfBirth has a @NotNull annotation so null pointers shouldn't be a problem here.